### PR TITLE
Converting Pages To Ant Design Template

### PR DESCRIPTION
**Pull request by ShariqAli-Dev**
### What work was done? 
- Converted Cases Page to Ant Design Layout
- Converted Case-View Page to Ant Design Layout
- Refactored Layout styling in .less 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -51,16 +48,13 @@ function App() {
 
   return (
     <Security {...config} onAuthRequired={authHandler}>
-      {/* hides the Navigation menu when at the login page, since the login page's purpose is singular */}
```
Comment (stkirk) [inline src/index.js:54]: This deletion would satisfy the bug fix for https://trello.com/c/h6sjGaeD/60-bugfix-with-new-layout-template-for-pages-navheader-component-is-rendered-twice
and we could close the PR Gabe made for that, right?

**Review by stkirk — DISMISSED** [review]
Great job getting these existing pages formatted to the new template. May require a rebase and resolving some merge conflicts in BreadCrumbs and src/index.js depending on the order we merge current open PRs

File: src/components/pages/CaseView/RenderCaseView.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+        <Footer className="Footer">
+          Family Promise of Spokane ©2022 Created by BloomTech Labs
+        </Footer>
+      </Layout>
+    </div>
+  );
+};
+
+export default RenderCaseView;
```
Comment (chavionjackson) [inline src/components/pages/CaseView/RenderCaseView.js:38]: nice job converting to ant design

**Review by wild-kard — APPROVED** [review]
Good work on refactoring the pages to feature the templates from antd.

I think this out of the 3 PR's we currently have open, this one should be merged AFTER shane's since this PR will likely generate a lot of merge conflicts. We can rebase Steve's PR from there since it's fairly small.

**Review by stkirk — APPROVED** [review]
changes look good here

**Review by Rex-1031 — APPROVED** [review]
Good work

**Review by JohnRyanB — APPROVED** [review]
Very nice PR. Thank you for getting the styling in line.

**Pull request merged**